packages/uui-ref-node-document-type/lib/uui-ref-node-document-type.story.ts

```diff
@@ -65,7 +67,7 @@ CustomIcon.parameters = {
 <uui-ref-node-data-type
   name="Product Page"
   alias="productPage">
-  <uui-icon slot="icon" name="shopping-basket-alt"></uui-icon>
+  <uui-icon slot="icon" name="colorpicker"></uui-icon>
```

Comment:
It should probably br a product icon the demonstrate the entity type? E.g. icons like tag, t-shirt og similar?

Reply:
Hi @bjarnef, we are a little limited in terms of built-in icons in the UI library. They are very generic, but feel free to browse around the uui-icon-registry-essential package and pick icons for a new PR if you'd like :-)

Reply:
@iOvergaard okay, I guess these icons are registreredes by default: https://github.com/umbraco/Umbraco.UI/tree/v1/dev/packages/uui-icon-registry-essential/lib/svgs

Reply:
Yes, those are "essential" for the UI library. I expect the backoffice will provide its own icon registry when that time comes.